### ISSUE TITLE
Preserve image location in markdown-document

### ISSUE THREAD

**Issue by nameless66** [wormi4ok/evernote2md#48]
Hello,

can you preserve the image location in the markdown document? All images are inserted at the end of the md-file.

Thanks in advance

**Comment by wormi4ok** [comment]
Hello @nameless66, thanks for reaching out! It should preserve the place of the image in the document. I've tested the latest version and didn't find any regressions. 
Maybe you could share an example note that has this issue so I can take a look? You can send it via e-mail or upload it here.

**Comment by nameless66** [comment]
Hi,
here is my document (enex) + generated notes folder. I used image-1.png two times in the Evernote-Doc, so one time export was correct.

But the pictures are inserted as text at the end of the MD-File.

Command was:
```
evernote2md Test\ evernote2md.enex
```

Brew versions (on Mac):
```
brew --version
Homebrew 3.0.11

brew info evernote2md
evernote2md: stable 0.16.0 (bottled)
```

[evernote2md.zip](https://github.com/wormi4ok/evernote2md/files/6277662/evernote2md.zip)

**Comment by wormi4ok** [comment]
Thanks for extensive information! This actually looks like a bug, I'll see what I can do in the next few days.

**Comment by wormi4ok** [comment]
I've released a patch version that should fix this issue - [v0.16.1](https://github.com/wormi4ok/evernote2md/releases/tag/v0.16.1). It will become available via `brew upgrade evernote2md` soon.

Please, let me know if the problem still exists.
